Feature-Maquetacion HTML y CSS  pokemon sort card  con BEM y refactorizacion archivos index.html style.css y card information

Johnbedoya1426 opened the pull request:
# HTML para proyecto pokemon

En este PR hicimos lo siguiente:

- Se maqueta con su HTML Y CSS el archivo pokemon sort
- Inicio de BEM en el archivo pokemon sort
- Se refactoriza el HTML y CSS de Index.HTML
- se refactoriza el HTML Y CSS de card information pokemon
- Vistas responsive  

-Index.HTML

| Desktop   | ipad | movil
| ---      | ---       |---
|<img width="1618" alt="Screenshot 2023-03-16 at 12 35 08 PM" src="https://user-images.githubusercontent.com/108825873/225706890-43b49392-d4a7-432c-b52d-49de2ead08b5.png"> | <img width="859" alt="Screenshot 2023-03-16 at 12 35 34 PM" src="https://user-images.githubusercontent.com/108825873/225707036-dacab2e3-aab7-4657-bdde-2f4a0d1bee29.png"> |<img width="590" alt="Screenshot 2023-03-16 at 12 35 57 PM" src="https://user-images.githubusercontent.com/108825873/225706063-4fa43580-6c57-4276-b8de-df8969845dce.png"> |

Card information Pokemon.HTML

| Desktop   | ipad | movil
| ---      | ---       |---
|<img width="1576" alt="Screenshot 2023-03-16 at 12 53 27 PM" src="https://user-images.githubusercontent.com/108825873/225727675-a4eec0d9-a1ed-46e4-bc08-8c5ecdf1537d.png"> | <img width="877" alt="Screenshot 2023-03-16 at 12 53 44 PM" src="https://user-images.githubusercontent.com/108825873/225727784-539b467b-7550-405a-ba85-d0809446e840.png"> |<img width="524" alt="Screenshot 2023-03-16 at 12 53 59 PM" src="https://user-images.githubusercontent.com/108825873/225727905-76e704db-947d-41c8-a1b6-4cf3e76e9f7e.png"> |

Pokemon card sort.HTML


| Desktop   | ipad | movil
| ---      | ---       |---
|<img width="904" alt="Screenshot 2023-03-16 at 12 56 12 PM" src="https://user-images.githubusercontent.com/108825873/225729179-97f98ad2-1f57-4992-8320-fe6fbded48e8.png"> | <img width="904" alt="Screenshot 2023-03-16 at 12 56 12 PM" src="https://user-images.githubusercontent.com/108825873/225729179-97f98ad2-1f57-4992-8320-fe6fbded48e8.png"> |<img width="904" alt="Screenshot 2023-03-16 at 12 56 12 PM" src="https://user-images.githubusercontent.com/108825873/225729179-97f98ad2-1f57-4992-8320-fe6fbded48e8.png"> |

Johnbedoya1426 committed:
initial commit

Johnbedoya1426 committed:
completing layout

lopezquekk commented:
Falta detalle del PR adjuntar videos y descripcion de lo que hace

lopezquekk commented on cardInformation.html:
```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en" id="card-information">
```
porque este ID?

lopezquekk commented on cardInformation.html:
```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en" id="card-information">
+<head>
+    <meta charset="UTF-8">
```
Todas las etiquetas se deben cerrar

Johnbedoya1426 replied:
la etiqueta cierra abajo antes de cerrar el documento

lopezquekk replied:
```
 <meta http-equiv="X-UA-Compatible" content="IE=edge">
 <meta charset="UTF-8">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
    <link rel="stylesheet" href="style.css">
```
    
    Ninguna de estas lo hace

Johnbedoya1426 replied:
HTML5 no permite el uso de etiquetas auto-cerradas
HTML5 y los diferentes navegadores tienen una gran capacidad para adaptarse a errores de sintaxis en el código, pero eso no significa que sea código válido.

lopezquekk commented on cardInformation.html:
```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
```
El nombre del documento no es descriptivo, que hace?

lopezquekk commented on cardInformation.html:
```diff
@@ -0,0 +1,83 @@
+                    <h2>Weight</h2>
+                </div>
+                <img src="./assets/Divider.svg" alt="divider">
+                <div class="height-information">
```
Porque las clases son tan especificas?, se pueden hacer reusables

lopezquekk commented on cardInformation.html:
```diff
@@ -0,0 +1,83 @@
+                    </div>
+                    <h2>Weight</h2>
+                </div>
+                <img src="./assets/Divider.svg" alt="divider">
```
El divider se debe hacer con CSS

Johnbedoya1426 replied:
arreglado

lopezquekk commented on cardInformation.html:
```diff
@@ -0,0 +1,83 @@
+                <img src="./assets/Divider.svg" alt="divider">
+                <div class="height-information">
+                    <div class="description-organize">
+                        <img src="./assets/VectorHeight.svg" alt="altura">
```
Con CSS

Johnbedoya1426 replied:
arreglado

lopezquekk commented on cardInformation.html:
```diff
@@ -0,0 +1,83 @@
+                </div>
+                <img src="./assets/Divider.svg" alt="divider">
+                <div class="moves-information">
+                    <p>Ability 1</p>
```
Esto no se podria hacer dinamico

Johnbedoya1426 replied:
arreglado

lopezquekk commented on cardInformation.html:
```diff
@@ -0,0 +1,83 @@
+                </div>
+             </div>
+             <div class="description-pokemon">
+                <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit.
```
no deberia existir saltos de linea en HTML

Johnbedoya1426 replied:
arreglado

lopezquekk commented on cardInformation.html:
```diff
@@ -0,0 +1,83 @@
+             </div>
+             <div class="stats-container">
+                <div class="stats">
+                    <h3 class="stat-hp">HP</h3>
```
Clases especificas que no se pueden volver dinamicas, cambiar esto

Johnbedoya1426 replied:
arreglado

lopezquekk commented on cardInformation.html:
```diff
@@ -0,0 +1,83 @@
+                    <h3 class="stat-spd">SPD</h3>
+                </div>
+                <div class="divider-stats">
+                    <img src="./assets/DividerLarge.svg" alt="divider">
```
Ningun Divider debe ser una image

Johnbedoya1426 replied:
arreglado

lopezquekk commented on index.html:
```diff
@@ -0,0 +1,77 @@
+        </div>
+        <div class="search-container">
+            <div class="search-button-container">
+                <button class="search-button">
```
Button no deberia cubrir el input

Johnbedoya1426 replied:
arreglado

lopezquekk commented on index.html:
```diff
@@ -0,0 +1,77 @@
+            </div>
+            <div class="card-pokemon">
+                <p>#999</p>
+                <figure class="container-img">
```
Porque es figure?

Johnbedoya1426 replied:
Un contenedor para que contenga la imagen y se pueda manipular

lopezquekk commented on index.html:
```diff
@@ -0,0 +1,77 @@
+            <div class="card-pokemon">
+                <p>#999</p>
+                <figure class="container-img">
+                    <img src="./assets/imagePokemon.svg" alt="img">
```
Hya imagenes en mayusculas iniciales y en minusculas iniciales, unificar el formato, preferiblemente, minusculas  ya que no son componentes

Johnbedoya1426 replied:
arreglado

lopezquekk commented on style.css:
```diff
@@ -0,0 +1,312 @@
+* {
```
Que reseter usaste?, debe ser uno mas riguroso para que sea compatible con otros navegadores

Johnbedoya1426 replied:
arreglado

lopezquekk commented on style.css:
```diff
@@ -0,0 +1,312 @@
+    margin: 24px;
+}
+.title-container img {
+    left: 0.51%;
```
cambiar CSS a rem o un manejo mejor de proporciones, no usar posiciones con porcentajes

Johnbedoya1426 replied:
arreglado

lopezquekk commented on style.css:
```diff
@@ -0,0 +1,312 @@
+    font-weight: 700;
+    font-size: 24px;
+    line-height: 32px;
+    margin-left: 1.2rem;
```
Combinando rem con px puede ser una mala practica, unificar

lopezquekk commented:
Fuentes no se ven bien
Espaciado está mal no llena la vista completa
La vista de detalle está mal

<img width="600" alt="Screenshot 2023-03-09 at 11 00 59 AM" src="https://user-images.githubusercontent.com/2974179/224081753-77f9fe3c-00e5-4ea5-bad3-b3cc06aeb4a6.png">
<img width="600" alt="Screenshot 2023-03-09 at 11 00 09 AM" src="https://user-images.githubusercontent.com/2974179/224081763-45e6a1b3-c394-4a30-8f96-4870230c3145.png">

Johnbedoya1426 committed:
code review

Johnbedoya1426 commented:
![This is an image](
![Screenshot 2023-03-09 at 5 47 56 PM](https://user-images.githubusercontent.com/108825873/224182086-e3eff58e-1c3c-4ff4-a266-594825316b81.png)
![Screenshot 2023-03-09 at 5 48 10 PM](https://user-images.githubusercontent.com/108825873/224182092-1f3541e4-029a-41dc-97aa-f42ddb7b340c.png)
![Screenshot 2023-03-09 at 5 48 33 PM](https://user-images.githubusercontent.com/108825873/224182099-309443c9-942c-4a20-a543-652444148c1d.png)
![Screenshot 2023-03-09 at 5 48 46 PM](https://user-images.githubusercontent.com/108825873/224182102-9f73985d-de2d-4627-9e1f-8d13e94968af.png)
![Screenshot 2023-03-09 at 5 51 20 PM](https://user-images.githubusercontent.com/108825873/224182105-3ec01e95-b5ad-4afd-b10d-4dbb3d492cca.png)
![Screenshot 2023-03-09 at 5 47 45 PM](https://user-images.githubusercontent.com/108825873/224182080-ff549f38-3e00-4e02-8624-e23cad544357.png)

Johnbedoya1426 committed:
Pokemon card sort terminado

-se crea el card pokemon sort con BEM
-Se refactoriza el HTMLy CSS del achivo index
-Se refactoriza el HTMLy CSS del achivo card information
-se cambian los nombres de los iconos e imagenes
